File: UnitTestProject1/Helpers/NavigationHelper.cs
namespace UnitTestProject1
{
    public class NavigationHelper : HelperBase
    {
        private string baseURL;

        public NavigationHelper(AppManager manager, string baseURL)
            : base(manager)
        {
            this.baseURL = baseURL;
        }

        public void GoToHomePage()
        {
            driver.Navigate().GoToUrl(baseURL);
        }
    }
}

File: src/test/java/ru/itis/todoist/helpers/NavigationHelper.java
package ru.itis.todoist.helpers;

import ru.itis.todoist.AppManager;

public class NavigationHelper extends HelperBase {
    private final String baseUrl;

    public NavigationHelper(AppManager manager, String baseUrl) {
        super(manager);
        this.baseUrl = baseUrl;
    }

    public void goToHomePage() {
        driver.navigate().to(baseUrl);
    }
}
